src/test/SumsTests.scala

```diff
@@ -96,25 +96,30 @@ class SumsTests extends munit.FunSuite:
     assertEquals(res.full, "magnolia1.tests.SumsTests.Color")
   }
 
-  test(
-    "report an error when an abstract member of a sealed hierarchy is not sealed"
-  ) {
+  test("report an error when an abstract member of a sealed hierarchy is not sealed") {
     val error = compileErrors("Show.derived[Parent]")
-    assert(
-      error contains "No given instance of type deriving.Mirror.Of[magnolia1.tests.SumsTests.Parent] was found for parameter x$1 of method derived in trait Derivation."
-    )
-    assert(
-      error contains "trait Parent is not a generic sum because its child trait BadChild is not a generic product because it is not a case class"
-    )
-  }
-
-  test(
-    "report an error when a concrete member of a sealed hierarchy is neither final nor a case class"
-  ) {
+    // TODO - maybe refactor the general derivation flow so that we can retain error messages from both mirrorful and mirrorless branches as they seem quite meaningful.
+    // For example errors could be:
+    // Error:
+    //  mirrorful approach failed with [...]
+    //  mirrorless approach failed with [...]
+    //
+    // assert(
+    //   error contains "No given instance of type deriving.Mirror.Of[magnolia1.tests.SumsTests.Parent] was found for parameter x$1 of method derived in trait Derivation."
+    // )
+    // assert(
+    //   error contains "trait Parent is not a generic sum because its child trait BadChild is not a generic product because it is not a case class"
+    // )
+    assert(error contains "Mirrrorful derivation not posiible, mirrorless derivation either not possible or not implemented yet.")
```

Comment:
at the very least we need to retain the type for which things fail

Reply:
refactored.